checkversion.sh

```diff
@@ -0,0 +1,7 @@
+CURRENT_VERSION=$(go version | awk 'NR==1{print $3}' | cut -c 3-)
+EXPECTED_MINIMAL_VERSION="1.13.0"
+echo "checking Go version..."
+if [[ $CURRENT_VERSION < $EXPECTED_MINIMAL_VERSION ]]; then
```

Comment:
You are comparing two strings, it will not work with: e.g EXPECTED_MINIMAL_VERSION="1.113.0", since 1.113.0 is going to be smaller than 1.13.0

This does not work with zsh only with bash

Reply:
You're right, thanks! I'm going to fix this on Monday!